grakn-test/src/test/java/ai/grakn/test/graql/reasoner/inference/GenealogyTest.java

```diff
@@ -106,12 +106,21 @@ public void testSpecificPerson(){
     }
 
     @Test
+    public void testFemale() {
+        String queryString = "match $x isa person has identifier $id has gender 'female';";
+        MatchQuery query = qb.parse(queryString);
+        QueryAnswers answers = new QueryAnswers(Reasoner.resolve(query, true).collect(Collectors.toSet()));
+        assertEquals(answers, Sets.newHashSet(qb.<MatchQueryAdmin>parse(queryString).results()));
+        assertTrue(answers.size() == 32);
```

Comment:
Use `assertEquals(32, answers.size())`, so if it fails we can see a clearer error message.